tests/linux/test_backend_utils.cpp

```diff
@@ -80,7 +80,7 @@ void simulate_qemuimg_resize(const mpt::MockProcess* process, const QString& exp
     EXPECT_THAT(args.at(2),
                 ResultOf([](const auto& val) { return mp::MemorySize{val.toStdString()}; }, Eq(expect_size)));
 
-    EXPECT_CALL(*process, execute).WillOnce(Return(produce_result));
+    EXPECT_CALL(*process, execute(300000)).Times(1).WillOnce(Return(produce_result));
```

Comment:
I wonder if this is a bit too specific. Do we really want to check the exact timeout? If so, it should probably be an imported constant. But I think this could just check `execute(_)`.

Reply:
Made this into a `mp::backend::image_resize_timeout` constant. We're likely to turn this into a setting at some point.